hepcrawl/utils.py

```diff
@@ -69,6 +71,14 @@ def collapse_initials(name):
     return name
 
 
+def split_fullname(author):
```

Comment:
I realized that this function is not ideal, it does not preserve the information in the original string via the comma. 
```python
In [5]: split_fullname("Doe, John Magic")
Out[5]: ('Doe', 'John Magic')  # correct

In [6]: split_fullname("Doe Boe, John Magic")
Out[6]: ('Doe', 'Boe John Magic') # Wrong
```

Can you add test cases to `test_utils.py` that makes sure that the following is correct:
```python
In [5]: split_fullname("Doe, John Magic")
Out[5]: ('Doe', 'John Magic')

In [6]: split_fullname("Doe Boe, John Magic")
Out[6]: ('Doe Boe', 'John Magic')
```